fast/stages/02-networking-nva/test-resources.tf

```diff
@@ -0,0 +1,246 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description temporary instances for testing
+
+# Untrusted (Landing)
+
+module "test-vm-landing-untrusted-ew1-0" {
+  source     = "../../../modules/compute-vm"
+  project_id = module.landing-project.project_id
+  zone       = "europe-west1-b"
+  name       = "test-vm-lnd-unt-ew1-0"
+  network_interfaces = [{
+    network    = module.landing-untrusted-vpc.self_link
+    subnetwork = module.landing-untrusted-vpc.subnet_self_links["europe-west1/landing-untrusted-default-ew1"]
+    alias_ips  = {}
+    nat        = false
+    addresses  = null
+  }]
+  tags                   = ["ew1", "ssh"]
+  service_account_create = true
+  boot_disk = {
+    image = "projects/debian-cloud/global/images/family/debian-10"
+    type  = "pd-balanced"
+    size  = 10
+  }
+  metadata = {
+    startup-script = <<EOF
+      apt update
+      apt install iputils-ping bind9-dnsutils
+    EOF
+  }
+}
+
+# TODO: comment all ew4 machines before pushing
```

Comment:
Are some of these supposed to be removed? (they look like a lot of VMs for testing) :)

Reply:
Not removed, but commented out by default. These will be commented as soon as we'll be (almost) ready for the merge :)

Reply:
Done. New commit coming.